chia/full_node/full_node.py

```diff
@@ -354,11 +354,11 @@ async def initialize_weight_proof(self):
         if peak is not None:
             await self.weight_proof_handler.create_sub_epoch_segments()
 
-    def set_server(self, server: ChiaServer):
+    def set_server(self, server: ChiaServer) -> None:
         self.server = server
-        dns_servers = []
+        dns_servers: List[str] = []
+        network_name = self.config["selected_network"]
```

Comment:
This can now raise an unconsumed exception.  That's ok?

Reply:
yes